packages/entities/entities-routes/cypress.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'cypress'
```

Comment:
Can you help me understand why you’re adding a completely new cypress component testing setup that replicates the functionality already provided in the repository?

Reply:
Removed